### PR TITLE
improvements in the usage of constraint layout and solved issue

### PR DESCRIPTION
**Pull request by MarcosCGlobant**
Some improvements in the constraints of the objects on the activity layout.

Added a scrollbar so the application can be fully seen on smaller devices. 

Solved an issue when doing operations with just a .(dot) as a value, now is not allowed. In the GIF i show what happen pressing equals with a .(dot) as an operator on the first and the second number.

Change buttons to not be all-caps from the get go.

constraint and dot

### CODE REVIEW

File: app/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,177 +1,248 @@
...
+            app:layout_constraintBottom_toTopOf="@id/add_button"
+            app:layout_constraintEnd_toStartOf="@id/number_four"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/clear_button" />
+
+        <Button
+            android:id="@+id/number_one"
+            style="@style/numberButtonStyle"
+            android:layout_width="wrap_content"
+
```
Comment (SartiMau) [inline app/app/src/main/res/layout/activity_main.xml]: remove this extra line pls

**Review by SartiMau — APPROVED** [review]

File: app/app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (MarcosCGlobant) [reply app/app/src/main/res/layout/activity_main.xml]: done, thanks

**Review by SartiMau — APPROVED** [review]

File: app/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,177 +1,247 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/activity_main"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/main_background"
-    android:orientation="vertical">
-
-    <TextView
-        android:id="@+id/calculation_label"
+    android:layout_width="fill_parent"
```
Comment (mauroLonghi) [inline app/app/src/main/res/layout/activity_main.xml]: Maros, this state of the property now is deprecated, so you need to change to match or wrap content.

Comment (MarcosCGlobant) [reply app/app/src/main/res/layout/activity_main.xml]: Done, Maro thanks

**Review by BQuinteros — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**